src/Goteo/Model/LegalDocumentType/CIF.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace Goteo\Model\LegalDocumentType;
+
+class CIF extends LegalDocumentType {
```

Comment:
It's missing the import of the extended class:

`use Goteo\Model\LegalDocumentType;`